src/App.js

```diff
@@ -1,12 +1,43 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import Button from './Components/Button';
+import AddTask from './Components/AddTask';
 import './App.css';
 
 function App() {
+  const [activeButton, setActiveButton] = useState(false);
+  const [tasks, setTasks] = useState([]);
+
+  const handleClick = (button) => {
+    setActiveButton(button);
+  };
+
+  const addTask = (task) => {
+    setTasks((prevTasks) => [...prevTasks, task]);
+  };
+
+  const isAddButtonActive = activeButton === "add";
+  const isViewButtonActive = activeButton === "view";
+
   return (
-    <div className="App">
-      
+    <div className="todo-container">
+      <h1>TODO APP</h1>
+      <Button
+        label="Add Task"
+        onClick={() => handleClick("add")}
+        backgroundColor="#1a5e66"
+        isActive={ isAddButtonActive }
+        isDisabled={false}
```

Comment:
There is no need to pass `isDisable` props here because the default value of isDisabled is false.